src/components/interfaces/HMI_API.xml

```diff
@@ -4783,6 +4783,27 @@
     </function>
     <function name="DialNumber" messagetype="response">
     </function>
+    <function name="OnResetTimeout" messagetype="notification">
+      <description>
+        HMI must send this notification to SDL for method instance for which timeout needs to be reset
+      </description>
+      <param name="requestID" type="Integer" minvalue="0" maxvalue="65535" mandatory="true">
```

Comment:
Could you please explain why the `maxvalue` is set to 65535 for this parameter

Reply:
@ShobhitAd I think it is mostly for documentation purposes to define the border values of uint16 type. Similar to [this](https://github.com/smartdevicelink/rpc_spec/blob/d4990d8da044a0b465493449317f22ccedf949bb/MOBILE_API.xml#L1230) line as an example, I believe this is mostly a common practice to define min/max values for such parameters.